src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,29 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
+
+- For ASP.NET Core 6, .NET 6
+- For ASP.NET Core 5, .NET 5
+- For ASP.NET Core 3, .NET Core 3.1
+- For ASP.NET Core 2, .NET Standard 2.0 and .NET 4.6.1+
+
+Registration of NLog.Web.AspNetCore in the NLog.config File
+
+```xml
+	<!-- enable ASP.NET layout renderers -->
+	<extensions>
+		<add assembly="NLog.Web"/>
```

Comment:
The xml-code still says NLog.Web and not NLog.Web.AspNetCore

Reply:
typing too fast...fixed again

Reply:
And also change `<add assembly="NLog.Web"/>` to `<add assembly="NLog.Web.AspNetCore"/>

Reply:
fixed